packages/run-protocol/src/vpool-xyk-amm/singlePool.js

```diff
@@ -10,8 +10,8 @@ import {
 /**
  * @param {ZCF} zcf
  * @param {XYKPool} pool
- * @param {() => bigint} getProtocolFeeBP - retrieve governed protocol fee value
- * @param {() => bigint} getPoolFeeBP - retrieve governed pool fee value
+ * @param {() => bigint} getProtocolFeeBP - Retrieve governed protocol fee value
+ * @param {() => bigint} getPoolFeeBP - Retrieve governed pool fee value
```

Comment:
Yes, it doesn't do any dash removal. That was me manually to be consistent with the places we don't use dashes and because the new capitalization provides a distinguishing feature when syntax highlighting isn't available.